inspect-traffic/compare-traffic.md

```diff
@@ -74,6 +74,7 @@ The **Compare Sessions (BETA)** tab provides the following options to customize
 - **Match sessions by**&mdash;Highlights unmatched sessions with default criteria set to match by **URL**. The possible match options are as follows:
     - **URL** match to highlight sessions with different URLs.
     - **Order** match session by their order in their ID column. The ID is the Fiddler Everywhere unique identification for each captured session.
+    - **Path** match to highlight sessions with different paths (but the same host).
```

Comment:
It's the other way around, same paths but different hosts. (also we ignore query and fragment from the URL)
PS: The wording in this article is a bit strange since we don't exactly highlight sessions, we match them to be on the same row in the 2 groups.